src/main/java/io/github/anthorx/parquet/sql/api/SQLParquetReader.java

```diff
@@ -93,26 +93,18 @@ public MessageType getSchema() {
   }
 
   public Record read() throws IOException {
-    if (currentParquetReader == null) {
-      return readFromNextParquetReader();
-    }
-
-    Record result = currentParquetReader.read();
+    Record resultParquetReader = null;
```

Comment:
resultRecord or recordResult is more appropriate here as this variable is not a parquetReader object but a Record one.

Reply:
Right so I fixed it (and I squashed all commits and did a push force)